Minor typos and grammar

NickLocke opened the pull request:
Fixed minor typos and grammar.

NickLocke committed:
Update laser_usage.blade.php - spelling.

NickLocke committed:
Update zone_occupants.blade.php - grammar.

NickLocke committed:
Update index.blade.php - spelling and grammar.

NickLocke committed:
Update index.blade.php - fixed typos.

NickLocke committed:
Update membershipComplete.blade.php - Removed Google Groups

NickLocke committed:
Update rejected.blade.php - grammar (singular/plural)

AaronJackson commented on resources/views/emails/membership/membershipComplete.blade.php:
```diff
@@ -16,9 +16,6 @@
 SSID: {{ $wifiSsid }}  
 Pass: {{ $wifiPass }}  
 
-Our Google Group is where a lot of online discussion takes place:  
```
```
@if ($groupLink)
Our Google Group is where a lot of online discussion takes place: 
{{ $groupLink }}
@endif
```

NickLocke committed:
Update membershipComplete.blade.php - selective groups

NickLocke committed:
Update membershipComplete.blade.php - remove hardcoded NH and make Google Groups conditional.

NickLocke committed:
Update membershipReinstated.blade.php - Made Google Group conditional and fixed a couple of hardcoded NH references.

NickLocke committed:
Update membershipRevoked.blade.php - grammar tweak

NickLocke committed:
Update membershipRevokedDueToUnderPayment.blade.php - grammar tweak

NickLocke committed:
Update interestRegistered.blade.php - Google and Discord [see extended description]

Made Google Groups conditional and added Discord. **Please check that the Discord bit had been done properly, and doesn't need changes elsewhere [I need to learn about Features]**

This one will address JonW's original concern about Discord not being mentioned in the Interest Registered email.

dpslwk commented on resources/views/emails/interestRegistered.blade.php:
```diff
@@ -12,8 +12,15 @@
+
+@feature('discord')
+Discord is used for members to chat online. You can join the {{ config('branding.space_type') }} Discord at:  
+{{ $discordHTML }}  
```
would also be best to use `@if($discordHTML)` over `@feature` since the check should be is the template being passed the any link (which is set via a Meta) now if the feature is enabled on the deployment

might be worth updating all email templates to match for `slackHTML`, `groupLink` and `discordLink`

Nottinghack is not the only deployment of HMS and and other do still use slack

NickLocke replied:
Amendments as suggested above have been made.

dpslwk reviewed (changes requested):
see comments

NickLocke committed:
Added discordHTML

NickLocke committed:
Use if rather than feature for Discord as suggested by Matt.

Updated Discord feature display logic in email template.

NickLocke committed:
Added more detail to the Discord message

As the Discord message now appears in the general Membership channel, rather than the private team channel, it makes sense to have a bit of descriptive text to make it clearer.
